pom.xml

```diff
@@ -72,11 +72,11 @@
             <version>1.16.5-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
+       <!-- <dependency>

```

Comment:
If we're removing it can we remove rather than comment out please.

Reply:
removed